roles/contiv_network/templates/netplugin.j2

```diff
@@ -1 +1,6 @@
-NETPLUGIN_ARGS=""
+{# If an interface is specified for netplugin, use it #}
+{% if netplugin_if is defined %}
+NETPLUGIN_ARGS='-docker-plugin -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
+{% else %}
+NETPLUGIN_ARGS='-docker-plugin -vtep-ip {{node_addr}}'
```

Comment:
Yes I saw the issue in netplugin. What I meant here was that ansible needs to fail early too if `netplugin_if` is not defined as there is no good default for `netplugin_if`

The current logic will hide the problem and let's ansible move on which will beat the purpose of catching error conditions early on.

The fix for netplugin repo will help folks who are trying to start netplugin themselves, which will be rare I think once we have cluster/installer working.